rotkehlchen/tests/utils/blockchain.py

```diff
@@ -679,3 +682,36 @@ def mock_arbitrum_one_has_activity(account):
         side_effect=mock_arbitrum_one_has_activity,
     ))
     return stack
+
+
+def maybe_modify_rpc_nodes(
```

Comment:
abstracted this out from `tests/fixtures/blockchain.py` since this was missing from rotkehlchen_api_server fixture and needed to also be called there.